src/sgame/sg_main.cpp

```diff
@@ -2991,9 +2991,20 @@ void G_RunFrame( int levelTime )
 void G_PrepareEntityNetCode() {
```

Comment:
My point is that the exported state pre-HandlePrepareNetcode can be completely different for whatever reason.